build.gradle

```diff
@@ -128,7 +128,7 @@ jacocoTestReport {
     }
 }
 
-String version = '8.0.0'
+String version = '8.0.1'
```

Comment:
Should be 8.1.0 as this is adding a new lib.

Reply:
Updated. Thanks.